ui/src/app/pages/admin/user/admin-user-egress-bypass.tsx

```diff
@@ -24,28 +24,31 @@ interface Props {
 }
 
 export const AdminUserEgressBypass = (props: Props) => {
-  const [startTime, setStartTime] = useState<CalendarValueType>(null);
+  const [startTime, setStartTime] = useState(new Date());
   const [bypassDescription, setBypassDescription] = useState('');
   const [apiError, setApiError] = useState(false);
   const [bypassWindowsList, setBypassWindowsList] = useState<
     EgressBypassWindow[]
   >([]);
-  const [reload, setReload] = useState(true);
 
-  useEffect(() => {
+  const loadEgressWindows = () => {
     const { userId } = props;
     userAdminApi()
       .listEgressBypassWindows(userId)
       .then((res) => setBypassWindowsList(res.bypassWindows));
-  }, [reload]);
```

Comment:
I think this was triggering twice as often as intended: false -> true and true -> false.  Adding an `if()` check here would have fixes this too, but I think this new version better expresses our intention.